packages/paste-website/src/pages/primitives/listbox-primitive/index.mdx

```diff
@@ -0,0 +1,322 @@
+
+<Callout variant="warning" marginY="space70">
+  <CalloutHeading as="h3">Before you roll your own listboxes...</CalloutHeading>
+  <CalloutText>
+    We strongly suggest that all components built on top of this primitive get reviewed by the Design Systems team and
+    go through the UX Review process to ensure an excellent experience for our customers.
+  </CalloutText>
+</Callout>
+
+## Examples
+
+### Selection
+
+To make items inside the Listbox selectable, you can manage the selection state yourself and combine it with the state returned from the `useListboxPrimitiveState` hook. To do so, track which item is selected in a separate store of state. When rendering the Listbox, cross reference the rendered items with the selection state, and conditionally set `selected` on each `ListboxPrimitiveItem` that requires it.
+
+The `onSelect` event handler will fire whenever the item is clicked, or the spacebar or enter key is pressed. Use this to respond to your users' selection interactions.
+
+### Horizontal Listbox
+
+Our listbox is vertical by default. To change it to horizontal, specify the orientation when initializing the state with `useListboxPrimitiveState({orientation: 'horizontal'})`:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {horizontalExample}
+</LivePreview>
+
+### Listbox with groups
+
+Use `ListboxPrimitiveGroup` to create different groupings within the same listbox:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveGroup,
+    ListboxPrimitiveItem,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {groupsExample}
+</LivePreview>
+
+### Multiselect Listbox
+
+Our listbox is set for single selection by default. To change it to allow multiple selections, specify the variant as `multiple` on `ListboxPrimitive`:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+  }}
+  noInline
+  showOverflow
+>
+  {multipleExample}
+</LivePreview>
+
+### Dual Listboxes
+
+Listboxes can be used in any situation where a user needs to select items in a list or rearrange a list. One example is a UI where users need to move items between two lists like this:
+
+<LivePreview
+  scope={{
+    useListboxPrimitiveState,
+    ListboxPrimitive,
+    ListboxPrimitiveItem,
+    Box,
+    Button,
+    CheckboxCheckIcon,
+    PlusIcon,
+    MinusIcon,
+    Column,
+    Grid,
+    Stack,
+  }}
+  noInline
+  showOverflow
+>
+  {dualExample}
+</LivePreview>
+
+### Custom styling
+
+You can provide custom styling to the primitive listbox by utilizing the `as` prop on each component.
+
+The listbox primitive does not come with any styling, and thus you could mix the functionality of it with another component by using the _as_ prop. By doing so, you can get styling from another component, and listbox functionality from this primitive.
```

Comment:
Question (non-blocking): Why is the "as" prop italicized here? Should it be a lil code snippet?

Reply:
Yeah can update in a follow-up!